src/lib389/lib389/replica.py

```diff
@@ -1650,6 +1650,9 @@ def get_ruv(self):
                 escapehatch='i am sure')[0]
             data = ensure_list_str(ent.getValues('nsds50ruv'))
         except IndexError:
+            # There are no ruv elements, it's okay
+            pass
+        except ldap.NO_SUCH_OBJECT:
```

Comment:
Minor nitpick, you can unite the exceptions with `except (IndexError, ldap.NO_SUCH_OBJECT):` line as the results are the same anyway.